admin/resources/js/components/CmoAssetTable.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { useMemo } from "react";
+import { Column } from "react-table";
+import { GetCmoAssetsQuery } from "../api/generated";
+import { notEmpty } from "../helpers/util";
+import Table from "./Table";
+
+const CmoAssetTable: React.FC<GetCmoAssetsQuery> = ({ cmoAssets }) => {
+  const columns: Array<Column> = useMemo(
+    () => [
+      {
+        Header: "ID",
+        accessor: "id",
+      },
+      {
+        Header: "Key",
+        accessor: "key",
+      },
+      {
+        Header: "Name",
+        accessor: "name.en",
+      },
+      {
+        Header: "Description",
+        accessor: "description.en",
+      },
+    ],
+    [],
+  );
+
+  const memoizedData = useMemo(() => cmoAssets.filter(notEmpty), [cmoAssets]);
+
+  return (
+    <>
+      <Table data={memoizedData} columns={columns} />
+    </>
+  );
+};
+
+export default CmoAssetTable;
```

Comment:
Lets also define a component here which wraps CmoAssetTable and includes the urql hook. Both components should be exported, and I'm not sure which should be the default export. Maybe no default export?

Reply:
I've left out the default export and export each one in CMO Assets and Operational Requirements.